bigbluebutton-html5/imports/ui/components/tooltip/component.jsx

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import Tippy from 'tippy.js';
+import _ from 'lodash';
+import cx from 'classnames';
+
+const propTypes = {
+  title: PropTypes.string.isRequired,
+  position: PropTypes.oneOf(['bottom']),
+  children: PropTypes.element.isRequired,
+  dynamicTitle: PropTypes.bool
+};
+
+const defaultProps = {
+  position: 'bottom',
+  dynamicTitle: true,
+};
+
+class Tooltip extends Component {
+  constructor(props) {
+    super(props);
+
+    this.tippySelectorId = _.uniqueId('tippy-');
+    this.onShow = this.onShow.bind(this);
+    this.onHide = this.onHide.bind(this);
+    this.handleEscapeHide = this.handleEscapeHide.bind(this);
+    this.delay = [250, 100];
+  }
+
+  componentDidMount() {
+    const {
+      position,
+      dynamicTitle,
+    } = this.props;
+
+    const options = {
+      position,
+      dynamicTitle,
+      delay: this.delay,
+      onShow: this.onShow,
+      onHide: this.onHide,
+    };
+
+    this.tooltip = Tippy(`#${this.tippySelectorId}`, options);
+  }
+
+  onShow() {
+    document.addEventListener('keyup', this.handleEscapeHide);
+  }
+
+  onHide() {
+    document.removeEventListener('keyup', this.handleEscapeHide);
+  }
+
+  handleEscapeHide(e) {
+    if (e.keyCode !== 27) return;
+
+    const popper = this.tooltip.tooltips[0].hide();
```

Comment:
popper isn't being used, can this be changed to only this.tooltip.tooltips[0].hide(); and omit the const popper?